src/main/java/supersymmetry/api/block/BlockSinteringBrick.java

```diff
@@ -0,0 +1,53 @@
+package supersymmetry.api.block;
+
+import gregtech.api.block.VariantActiveBlock;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.EntityLiving;
+import net.minecraft.util.IStringSerializable;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+import org.jetbrains.annotations.NotNull;
+
+public class BlockSinteringBrick extends VariantActiveBlock<BlockSinteringBrick.SinteringBrickType> {
+
+    public BlockSinteringBrick() {
+        super(Material.ROCK);
```

Comment:
This should probably match whatever material minecraft's brick blocks use.

Reply:
Rock should be the correct one afaik

Reply:
Can't seem to find where in the game's code it assigns the material, but the wiki says it's supposed to be `Material.STONE`.
Since `Material.STONE` doesn't exist I think the correct one is `Material.ROCK`.